lib/internal/process/per_thread.js

```diff
@@ -148,6 +149,46 @@ function wrapProcessMethods(binding) {
     };
   }
 
+  const threadCpuValues = new Float64Array(2);
```

Comment:
I recommend moving this to C++ side and updating it. A similar implementation exist in node url

Reply:
Will look it up. Thanks.

Reply:
I tried to look it up in the code but I couldn't find it.
Do you mind linking a reference to the similar implementation so I can check it out?